test/ModuleManifest.Tests.ps1

```diff
@@ -1,3 +1,4 @@
+[System.Diagnostics.CodeAnalysis.SuppressMessage('PSUseDeclaredVarsMoreThanAssigments', '', Scope='*', Target='SuppressImportModule')]
```

Comment:
In general we exclude our C# unit tests from code analysis.  Should we do the same here - exclude Pester tests from script analysis?  I would be in favor of that.  Getting your module's script copasetic with PSSA can be enough of a challenge.  :-)  Or was this to get rid of a VSCode editor warning message about the rule?